webapi/_test_data/all_data/anything-data.ttl

```diff
@@ -7,806 +7,806 @@
 @prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
 @prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
 @prefix standoff: <http://www.knora.org/ontology/standoff#> .
-@prefix anything: <http://www.knora.org/ontology/anything#> .
+@prefix anything: <http://www.knora.org/ontology/0001/anything#> .
 
-<http://data.knora.org/0C-0L1kORryKzJAJxxRyRQ> a anything:Thing ;
+<http://rdfh.ch/0C-0L1kORryKzJAJxxRyRQ> a anything:Thing ;
```

Comment:
yes, you are right.

Reply:
but this is another discussion. I will do it as described [here](http://docs.knora.org/sphinx/03-knora-api-server/api_v2/knora-iris.html#id6)

Reply:
That is how we formatted our data too, so, so far, our prod data is compliant to the doc.

But our (Lausanne) next upgrade will be a big one, so I would be in favour to push as much changes as we want to push now and have smoother releases in the future.

Reply:
done

Reply:
Did you want to do this for `incunabula-data.ttl`, too?

Reply:
Oh, please no. This will take too long. The anything data is small, while incunabula is huge. I need to finish some things for Lukas before my vacation next week and after I'm back, prepare the production server, so I won't be working on Knora until June.